### DISCUSSION TITLE
Pass the values of hlsSegmentCount and hlsSegmentDuration in env variable.

### DISCUSSION THREAD

**Discussion by Prakash2101** [bluenviron/mediamtx#437] [Questions]
I'm running this application on kubernetes, so for hls streaming how can I pass the hlsSegmentCount and hlsSegmentDuration parameter in env variable? Can you share the correct way to pass the env variable for said parameter.

**Comment by aler9** [comment]
```
RTSP_HLSSEGMENTCOUNT=5
RTSP_HLSSEGMENTDURATION=10s
```

**Reply by Prakash2101** [reply]
Thanks 😊

[a comment by an automated account is not shown]
